Client/src/components/common/Course/index.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+import'./index.css'
+
+export default function CoursCard(props) {
+  const { titel, desc } =props
+  return (
+    <div className='CoursCard'>
```

Comment:
I think you used the`  Camel Case ` in naming your class'
So, Its better to follow one method for more professionalism

Reply:
converting to camel  case